lib/jobs/ansible-job.js

```diff
@@ -78,10 +95,23 @@ function ansibleJobFactory(
      * @memberOf AnsibleJob
      */
      AnsibleJob.prototype._cleanup = function cleanup() {
+        var self = this;
+        try {
+            var data = fs.readFileSync(self.outFileName, 'utf8');
```

Comment:
My meaning is that it's a problem that need be resolved as you mentioned sometimes the file content was not in the context.